Examples/Physics_applications/thomson_parabola_spectrometer/analysis.py

```diff
@@ -0,0 +1,79 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+from openpmd_viewer import OpenPMDTimeSeries
+from scipy.constants import c, eV
+
+mpl.use('Agg')
+mpl.rcParams.update({'font.size': 18})
+
+MeV=1e6*eV
+
+# some parameters from the inputfile
+Emax_hydrogen1_1 = 40*MeV
+Emax_carbon12_6 = 20*MeV
+Emax_carbon12_5 = 20*MeV
```

Comment:
```suggestion
Emax_carbon12_4 = 20*MeV
```
Shouldn't this be `Emax_carbon12_4` ?

Reply:
But in any case it seems that these variables are unused. Maybe you can comment them or delete them